doc/build-openbsd.md

```diff
@@ -21,23 +18,23 @@ pkg_add boost
 git clone https://github.com/bitcoin/bitcoin.git
 ```
 
+Optional: To build with Qt 5 support, you also need qt5 and protobuf:
+```bash
+pkg_add protobuf
+pkg_add qt5
+```
+
 See [dependencies.md](dependencies.md) for a complete overview.
 
-**Important**: From OpenBSD 6.2 onwards a C++11-supporting clang compiler is
-part of the base image, and while building it is necessary to make sure that this
-compiler is used and not ancient g++ 4.2.1. This is done by appending
-`CC=cc CXX=c++` to configuration commands. Mixing different compilers
-within the same executable will result in linker errors.
 
 ### Building BerkeleyDB
 
-BerkeleyDB is only necessary for the wallet functionality. To skip this, pass
-`--disable-wallet` to `./configure` and skip to the next section.
+BerkeleyDB is only necessary for the wallet functionality. To skip this,
+pass `--disable-wallet` to `./configure`.
```

Comment:
Why remove "and skip to the next section" here?

Reply:
The first sentence already states that this step only is required for the wallet functionality. I figured that we explain that "--disable-wallet" is how you disable wallet functionality. But adding "and skip to the next section" sounds like you should run ./configure with --disable-wallet AND skip to the next section (which tells you to run configure again?)